README.md

```diff
@@ -33,12 +33,28 @@ Instead, create your own new node project, and install the oss-mariner package v
    1.1. <https://github.com/indeedeng/Mariner/blob/master/examples/runFasterCode.ts>
 1. Comment out the existing line that imports mariner.
 1. Uncomment the line saying how mariner would normally be imported.
+1. Create a config.json file inside examples folder
+1. Include in the json file:
+
+```
+{
+    "numberOfReposPerCall": 600,
+    "labelsToSearch": [
+        "good first issue",
+        "help wanted"
+    ],
+    "inputFilePath": "examples/exampleData.json",
+    "outputFilePath": "examples/output.json"
+}
+
+```
+
+1. Create an exampleData.json file or copy it in from Mariner
+
 1. Convert the TypeScript code to JavaScript by
```

Comment:
I would like to edit this to: "Mariner supports TypeScript, but we don't have step-by-step instructions for the TypeScript example. For now, you can convert the runFasterCode.ts example file to JavaScript:"